README.md

```diff
@@ -63,279 +49,281 @@ Regardless of the input type used, all elements follow the same structure:
-	...
--	<span class="o-forms-input o-forms-input--radio-box">
-+	<span class="o-forms-input o-forms-input--radio-box o-forms-input--saving">
-		<div class="o-forms-input--radio-box__container">
-			<label>
-				<input type="radio" name="negative" value="Yes">
-				<span class="o-forms-input__label">Yes</span>
-			</label>
-			<label>
-				<input type="radio" name="negative" value="No"checked>
-				<span class="o-forms-input__label o-forms-input__label--negative">No</span>
-			</label>
-		</div>
--		<span class="o-forms-input__state"></span>
-+		<span class="o-forms-input__state o-forms-input__state--icon-only"></span>
+[_See radio buttons in the registry_](https://registry.origami.ft.com/components/o-forms#demo-round-styled-radio-inputs)
+
+##### Box radio inputs
+
+For a box-like, rectangular radio button replace the `o-forms-input--radio-round` modifier class with `o-forms-input--radio-box`.
+
+Box radio buttons may also, optionally represent a negative choose by applying the `o-forms-input__label--negative` modifier class to its label. This is useful for a yes/no input.
+
+Loading states are also supported by box radio buttons. We recommend using [`o-forms` JavaScript](#javascript) to add states to an existing form. However to add a state manually add an additional `o-forms-input__state` element and a state modifier class to the `o-forms-input` element, `o-forms-input--saving` or `o-forms-input--saved`.
+
+The below example shows a box style radio button with a positive "yes" and negative "no" option. The "no" option is checked and a saved state is shown.
+
+```html
+<div class="o-forms-field" aria-labelledby="example-group-title" role="group">
+	<span class="o-forms-title">
+		<span class="o-forms-title__main" id="example-group-title">Group Title</span>
+	</span>
+
+	<span class="o-forms-input o-forms-input--radio-round o-forms-input--saved">
+		<label>
+			<input type="radio" name="positive" value="Yes">
+			<span class="o-forms-input__label">Yes</span>
+		</label>
+		<label>
+			<input type="radio" name="negative" value="No" checked>
+			<span class="o-forms-input__label o-forms-input__label--negative">No</span>
+		</label>
+
+		<span class="o-forms-input__state"></span>
 	</span>
 </div>
 ```
 
-If you would like custom copy for the "saving" or "saved" state put it within the `o-forms-input__state` element, and add the modifier class `o-forms-input__state--custom`. _We recommend setting a custom label using the [setState method](#state) JS method instead of adding this markup manually._
+To show a state label with no text set the modifier class `o-forms-input__state--icon-only` on the `o-forms-input__state` state element. Or to use custom copy for the saving and saved states add the modifier class `o-forms-input__state--custom` and put your copy within the state element. However, as mentioned previously, we recommend setting a custom label using the [setState method](#state) JavaScript method instead of adding this markup manually.
```

Comment:
```suggestion
To show a state label with no text set the modifier class `o-forms-input__state--icon-only` on the `o-forms-input__state` state element. Or to use custom copy for the saving and saved states add the modifier class `o-forms-input__state--custom` and put your copy within the state element. However, as mentioned previously, we recommend setting a custom label using the [setState JavaScript method](#state) instead of adding this markup manually.
```